### PR TITLE
DISCO methods paper, initial attempt to address feedback from reviewers 2&3

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

This is a draft, I need to finish up these responses tomorrow.

### Suggested reviewers (optional)

@agitter tagging you just to loop you in that responses to most of the comments are in the works. It's not ready for review quite yet though.

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/60.methods.md
Comment (agitter) [inline content/60.methods.md:110]: Reviewer 3 suggested using footnotes for URLs, and I recommend that for URL citations as well.  The URL citations do not translate well to the conference LaTeX template and are quite difficult to fix manually.  I think we could keep URL citations for the small number of URLs that refer to an article with a specific author, title, and date (like `url:https://blogs.sciencemag.org/pipeline/archives/2021/07/19/too-many-papers`) and convert the rest.

Pandoc does support footnotes (see https://github.com/manubot/rootstock/issues/352).  We would need to see how they show up in the long HTML document and the converted tex file.

```suggestion
Authorship was determined based on the Contributor Roles Taxonomy [@url:https://casrai.org/credit].
```

File: content/60.methods.md
Comment (agitter) [inline content/60.methods.md:73]: Some initial text you can work from.  Maybe this would fit as a new paragraph following this one.

> We selected Manubot because it offers several advantages over comparable collaborative writing platforms such as Authorea, Overleaf, Google Docs, Word Online, or wikis [@doi:10.1371/journal.pcbi.1007128].
Citation-by-identifier ensures consistent reference metadata standards that would be difficult to maintain manually in a manuscript with dozens of authors and over 1,500 citations.
Manubot's pull request-based contribution model balances the goals of making the project open to everyone and maintaining scientific accuracy.
All contributions are reviewed, discussed, and formally approved on GitHub before text updates appear in the public-facing manuscript at <https://greenelab.github.io/covid19-review/>.
Continuous integration (CI) seamlessly combines author-produced text and figures with automatically generated and updated statistics and figures from external data sources and the manuscript's own content.
In addition, the authors who initially launched this project included Manubot developers who had prior successes using Manubot for massively open and traditional manuscript.

We could use `analyze-ms-stats/manuscript_stats.json` from #1019 to automate the exact number of references.  The last sentence is meant to acknowledge that we also selected Manubot in part because of familiarity and our existing bias toward it (driven by prior successes).

File: content/60.methods.md
Comment (agitter) [inline content/60.methods.md:262]: Here is an initial summary:

- CI scripts to regularly download external data sources, generate figures and statistics, and store them in a GitHub branch that is read when Manubot builds the latest manuscript
- Zotero extension to report more relevant clinical trial metadata from https://clinicaltrials.gov
- Manubot feature to support citing any Compact Uniform Resource Identifier, such as `clinicaltrials` or `ncbigene`
- Integration with scite to track retractions, corrections, and notices of concern
- CI spell-checking reports that post as pull request comments
- Improved support for Pandoc's LaTeX output
- Building individual sections of a larger manuscript as standalone documents alongside the complete manuscript

This feels redundant with the existing text.  Do we agree with the reviewer that the summary is helpful?

File: content/60.methods.md
Comment (agitter) [inline content/60.methods.md:291]: Let's consider additional limitations of the Manubot approach.  Ryan brought up a good one at https://github.com/greenelab/covid19-review/pull/947#pullrequestreview-649162037  Readers of the HTML manuscript cannot tell what parts of the paper are up to date and which are stale.  A savvy reader could use GitHub's blame feature to determine when each sentence or figure was last edited, but that information is not displayed in the rendered manuscript and doesn't distinguish between typo corrections and substantive updates.

A related idea is that we have automating figures and statistics that are summarized by static contributed text.  There is a risk that the data in the figure or statistics can change and the related text will be wrong or inconsistent.  We sometimes tried to work around this by writing "As of date XYZ" in the text, but the core limitation remains.

The pull request model can sometimes be tedious.  Adding even a single sentence requires making a pull request, waiting for an approval, and waiting for the CI build to finish.  That has deterred some colleagues from joining Manubot projects in the past.

There is no good way to provide general comments on a manuscript.  Comments can be added to pull requests, but only to the modified content.  Hypothesis comments can be added anywhere but are not easy to integrate back into the manuscript like pull request comments or suggestions.

Any other limitations we've faced?

**Review by agitter — COMMENTED** [review]
Thanks for starting on these updates @rando2.  Because we only have a few days to finish these revisions, should we work together in this pull request to make the remaining edits?

I know this is still a work in progress, but I added some initial thoughts and text you can merge in.

File: content/60.methods.md
Comment (rando2) [reply content/60.methods.md:299]: Another challenge is definitely that we haven't pivoted completely away from DOCX. I have been manually adding commits that people send me either in emails or in docx attachments (using --author). 

Also, there are silly issues like Casey's setting automatically remove EOL whitespace, so every time he opens a PR, it has a huge impact in terms of line numbers. We can fix this by automating this for every PR, but I haven't gotten to it!

Another one-- we don't have a good way to link up preprints with their published papers. I sometimes do this randomly while checking the text -- I'll open a preprint to confirm that I'm not changing the meaning with some edit, and then find that it has been published. I think we can also add this thanks to work David Nicholson has been doing.

File: content/60.methods.md
Comment (rando2) [reply content/60.methods.md:73]: I need to get that PR finalized -- will work on that next!

File: content/60.methods.md
Comment (rando2) [reply content/60.methods.md:283]: I think I need to move line 282 to the paragraph below to make this more clear, we cannot fix the pandemic with Manubot!

File: content/60.methods.md
Comment (rando2) [reply content/60.methods.md]: Yours is better than mine! I do like the idea of a table, because I have to keep looking through the text to jog my memory about the full list!

File: content/60.methods.md
Comment (rando2) [inline content/60.methods.md:269]: > * otero extension to report more relevant clinical trial metadata from https://clinicaltrials.gov
> 
> * Manubot feature to support citing any Compact Uniform Resource Identifier, such as `clinicaltrials` or `ncbigene`
> 
> * Integration with scite to track retractions, corrections, and notices of concern
> 
> * CI spell-checking reports that post as pull request comments
> 
> * Improved support for Pandoc's LaTeX output
> 
> * Building individual sections of a larger manuscript as standalone documents alongside the complete manuscript

File: content/60.methods.md
Comment (rando2) [inline content/60.methods.md:267]:
```suggestion
| - CI scripts to regularly download external data sources, generate figures and statistics, and store them in a GitHub branch that is read when Manubot builds the latest manuscript |
| - Zotero extension to report more relevant clinical trial metadata from https://clinicaltrials.gov |
| - Manubot feature to support citing any Compact Uniform Resource Identifier, such as clinicaltrials or ncbigene
| - Integration with scite to track retractions, corrections, and notices of concern |
| - CI spell-checking reports that post as pull request comments |
| - Improved support for Pandoc's LaTeX output |
| - Building individual sections of a larger manuscript as standalone documents alongside the complete manuscript |
```

File: content/60.methods.md
Comment (agitter) [reply content/60.methods.md]: We could follow up in that rootstock issue to discuss modifying 02.delete-me.md.  There is always a gray area about how much to demonstrate in that example document.  We try to not fully reproduce the pandoc documentation when the feature is pure pandoc but also need to demonstrate common use cases.

File: content/60.methods.md
Comment (agitter) [reply content/60.methods.md:302]: Other limitations from the reviews we could mention:
- We integrated with the Mount Sinai preprint reviews but any other relevant efforts (e.g. preLights).  There are other potential contributors who are/were reviewing COVID-19 preprints.  How could we build upon their existing efforts?
- scite slows down HTML load times (https://github.com/greenelab/covid19-review/issues/1015).  Could we load the badges only on request?
- Formal user study of Manubot versus other collaborative writing approaches, provide evidence as to how usable (or not) Manubot is for non-technical users.  Bias among current authors toward technical users (related to the point above about DOCX).
- Machine readable/actionable summaries of key information in the review manuscripts as a way to combat the infodemic (as opposed to creating yet another set of manuscripts to read)

File: content/60.methods.md
Comment (dhimmel) [reply content/60.methods.md]: Could rephrase to:

> Zotero translator to extract citation metadata from ClinicalTrials.gov.

Want to stress _citation_ metadata, since reader may not be familiar with Zotero.

Otherwise suggestion looks great. An impressive list of contributions!

File: content/60.methods.md
Comment (rando2) [reply content/60.methods.md:74]: Once #1034 merges, we should be able to make all of these dynamic. There are several places throughout the text where they come up.

File: content/60.methods.md
Comment (rando2) [reply content/60.methods.md:289]: Just wanted to remark that I'm in the process of reorganizing these paragraphs and adding ideas from the discussion here, and I'll push the changes later this morning!

**Comment by agitter** [comment]
Yesterday I pushed two commits to work on the figures and speed up the build time by disabling the pharmaceuticals docx build.

The footnotes look pretty good on the HTML and PDF.  However, we should be consistent about the order when we have a footnote and a citation.
image

We'll also need to test the footnotes in LaTeX.

File: content/60.methods.md
Comment (rando2) [reply content/60.methods.md:311]: I believe these are all addressed in the new draft text!

**Comment by rando2** [comment]
> Yesterday I pushed two commits to work on the figures and speed up the build time by disabling the pharmaceuticals docx build.
> 
> The footnotes look pretty good on the HTML and PDF. However, we should be consistent about the order when we have a footnote and a citation.
> image
> 
> We'll also need to test the footnotes in LaTeX.

Yikes, the footnote & citation combo is not very pretty! Which do you things looks better/worse?

**Comment by agitter** [comment]
> Yikes, the footnote & citation combo is not very pretty! Which do you think looks better/worse?

I think footnote before citation is the better of the two options, like 3 and 4 in the screenshot above.  Otherwise it looks like the footnote is modifying the citation.

**Review by agitter — APPROVED** [review]
These changes look good to me.  I'm approving now even if @rando2 still has minor changes to make.  I'd like to merge in this large batch of changes before I start final proofreading.

File: content/60.methods.md
Comment (rando2) [inline content/60.methods.md:262]:
```suggestion
While Manubot manuscripts are written in markdown, they can be rendered in several formats that provide different advantages for a variety of users.
```

File: content/60.methods.md
Comment (rando2) [inline content/60.methods.md:306]:
```suggestion
This asynchrony could potentially introduce incompatibility between the figures and the surrounding text.
```

File: content/60.methods.md
Comment (rando2) [inline content/60.methods.md:307]:
```suggestion
Similarly, in line with the collaboration-related challenges of the project, some authors returned to update their text, while others did not.
```

**Review by rando2 — COMMENTED** [review]
Fix spelling (I have a different branch checked out locally so working through github on this!)

**Review by rando2 — COMMENTED** [review]
Move footnotes before refs

File: content/60.methods.md
Comment (rando2) [inline content/60.methods.md:329]:
```suggestion
We are grateful to Josh Nicholson and Milo Mordaunt for their support with the scite plugin, to Milton Pividori for feedback on code, to David Nicholson for the suggestion and feedback to enable the reporting of the locations of spelling errors in the spell-checker tool, and to Consortium members Alex Lee and Christian Brueffer for feedback on the code and text, respectively.
```

File: content/60.methods.md
Comment (rando2) [inline content/60.methods.md:331]:
```suggestion
This research was supported by the Gordon and Betty Moore Foundation award GBMF 4552 (HMR, DSH, CSG), the National Institutes of Health award R01HG010067 (HMR, CSG), and the John W. and Jeanne M. Rowe Center for Research in Virology (AG).
```

File: content/60.methods.md
Comment (rando2) [inline content/60.methods.md:260]:
```suggestion
In our case, where the full project is nearly 150,000 words, this allows individual pieces to be shared more easily.
```

File: content/60.methods.md
Comment (rando2) [inline content/60.methods.md:282]:
```suggestion
Some challenges, however, arose in efforts to include an academically diverse set of authors, either due to the unique circumstances of the pandemic or as the result of challenges broadly associated with open publishing projects.
```

File: content/60.methods.md
Comment (rando2) [inline content/60.methods.md:284]:
```suggestion
Using pull requests as a tool for writing text is also unfamiliar to many or most scientists, and the review process can be slow, which might cause interested contributors to lose interest.
```

File: content/60.methods.md
Comment (rando2) [inline content/60.methods.md:294]:
```suggestion
With such a large amount of text, it is not possible to keep all sections of the manuscript up to date at all times.
```

File: content/60.methods.md
Comment (rando2) [inline content/60.methods.md:297]:
```suggestion
This asynchronicity could potentially introduce incompatibility between the figures and the surrounding text.
```

File: content/60.methods.md
Comment (rando2) [inline content/60.methods.md:305]:
```suggestion
As it stands, the integration of Compact Uniform Resource Identifier does make a step in this direction, as it could be used to extract relationships among clinical trials, genes, publications, and other entities.
Thus, the experience of using Manubot for a massive non-technical project has laid the foundation for future additions to enhance user experience and inclusivity.
```

**Review by rando2 — COMMENTED** [review]
Final read-through for tonight

File: content/60.methods.md
Comment (agitter) [inline content/60.methods.md:316]: Woah!  Where did the 3,000 commits come from?  Is this one of the stats you now track in a JSON file?  It looks like it distinguishes between commits associated with pull requests versus all 5,000 commits to master (which includes rootstock and non-content commits).

Comment (agitter) [reply content/60.methods.md:316]: Never mind, now I see `"manuscript_commits": 2886` in `covid19-review-stats.json`

**Pull request merged**
